pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -693,17 +743,24 @@ func buildSidecarOutboundTCPListenerOptsForPortOrUDS(listenerMapKey *string,
 	var destinationCIDR string
 	if len(listenerOpts.bind) == 0 {
 		svcListenAddress := listenerOpts.service.GetAddressForProxy(listenerOpts.proxy)
+		svcExtraListenAddress := listenerOpts.service.GetExtraAddressesForProxy(listenerOpts.proxy)
 		// We should never get an empty address.
 		// This is a safety guard, in case some platform adapter isn't doing things
 		// properly
 		if len(svcListenAddress) > 0 {
 			if !strings.Contains(svcListenAddress, "/") {
 				listenerOpts.bind = svcListenAddress
+				if len(svcExtraListenAddress) > 0 {
```

Comment:
you can just set it always, setting to empty is NOP, right?

Reply:
yes, set `exBind` directly

Reply:
done